src/blocks/dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC, forwardRef, useState } from 'react';
+import styled from 'styled-components';
+import * as RadixDropdown from '@radix-ui/react-dropdown-menu';
+
+import { DropdownProps } from './Dropdown.types';
+import { getDropdownPositionalCSS } from './Dropdown.utils';
+
+const RadixDropdownContent = styled(RadixDropdown.Content)<DropdownProps>`
+  /* Extra CSS props */
+  ${(props) => props.css || ''}
+`;
+
+const Dropdown: FC<DropdownProps> = forwardRef<HTMLButtonElement, DropdownProps>(
+  ({ overlay, trigger = 'click', children, dropdownPosition = 'bottom', ...props }, ref) => {
+    const [isOpen, setIsOpen] = useState(false);
+
+    const showDropdown = () => setIsOpen(true);
+    const hideDropdown = () => setIsOpen(false);
+    const toggleDropdown = () => setIsOpen(!isOpen);
+
+    const { ...cssProps } = getDropdownPositionalCSS(dropdownPosition);
```

Comment:
We don't need this now.  <RadixDropdownContent/> supports an align prop whose value can be `start`, `end` and `center`, I think for now we can use that.